khmer/_oxli/hllcounter.pxd

```diff
@@ -1,38 +1,47 @@
 from libcpp cimport bool
 from libcpp.memory cimport unique_ptr
-from libcpp.sting cimport string
+from libcpp.string cimport string
 from libcpp.vector cimport vector
 from libc.stdint cimport uint64_t
 
 from oxli_types cimport *
 from parsing cimport CpReadParser
 
+
+cdef extern from "oxli/oxli_exception_convert.hh":
+    cdef void oxli_raise_py_error()
+
+
 cdef extern from "oxli/hllcounter.hh" namespace "oxli":
     cdef cppclass CpHLLCounter "oxli::HLLCounter":
-        CpHLLCounter(double, WordLength)
-        CpHLLCounter(int, WordLength)
+        CpHLLCounter(double, WordLength) except +oxli_raise_py_error
```

Comment:
This allows capturing oxli exceptions in C++ and converting to appropriate Python exceptions. (But maybe we can use a shorter name...)